lisp/ess-r-mode.el

```diff
@@ -252,6 +252,11 @@ When t, loading a file into a namespaced will output information
 about which objects are exported and which stay hidden in the
 namespace.")
 
+(defvar ess-newest-R nil
```

Comment:
Shall we make this one internal and make it more explicit like `ess-r--newest-executable` or something?